src/ntrip_client/ntrip_client.py

```diff
@@ -145,26 +151,34 @@ def connect(self):
 
     # Some debugging hints about the kind of error we received
     known_error = False
+    error_code = self.ERROR_CODE_NO_ERROR
     if any(sourcetable in response for sourcetable in _SOURCETABLE_RESPONSES):
       self._logwarn('Received sourcetable response from the server. This probably means the mountpoint specified is not valid')
       known_error = True
+      error_code = self.ERROR_CODE_INVALID_MOUNTPOINT
```

Comment:
```suggestion
      error_flags = error_flags | self.ERROR_FLAG_INVALID_MOUNTPOINT
```
Switch to bitwise operations for setting the flag